src/rocker/templates/rmw_snippet.Dockerfile.em

```diff
@@ -4,13 +4,16 @@
 @[ if rmw ]@
 RUN \
   if [ -z "${ROS_DISTRO}" ]; then echo "ROS_DISTRO is unset cannot override RMW" ; exit 1 ; fi ;\
-  if dpkg -l @(' '.join(packages)) > /dev/null 2>&1; then \
-   apt-get update \
-   && DEBIAN_FRONTENT=non-interactive apt-get install -qy --no-install-recommends\
-     @(' '.join(packages)) \
-   && apt-get clean ;\
+@[for package in packages]@
+  apt-get update ;\
```

Comment:
Minor nit: doing the apt-get update and apt-get clean as setup and teardown to the loop would cut down on timing. It's not important while the loop size is usually 1.